_includes/install/php_2.2.md

```diff
@@ -1,2 +1,2 @@
-| 7.0.0&ndash;7.0.12 | 7.0.13&ndash;7.1.x | 7.2.0&ndash;7.2.11 |
-| ![Not supported]({{ site.baseurl }}/common/images/red-x.png) | ![Supported (2.1.2 and later)]({{ site.baseurl }}/common/images/green-check.png) | ![Supported (2.1.2 and later)]({{ site.baseurl }}/common/images/green-check.png) |
+| 7.0.0&ndash;7.0.12 | 7.0.13&ndash;7.1.0 | 7.2.0&ndash;7.2.11 |
```

Comment:
This should probably be a four-column table.

Change the heading of the second column to 7.0.13 - 7.0.x (green check)
Insert a new column with the heading of 7.1.x (green-check)
Change the last column heading to 7.2.x (red-x)

The tildes mean greater than or equal to, without incrementing the values to the left of the last period.

Don't take my word as gospel. Get an architect or senior developer to review